renderer.php

```diff
@@ -831,7 +836,7 @@ public function post_template($p) {
             if (empty($p->parentuserpic)) {
                 $byline = get_string('replybyx', 'hsuforum', $byuser);
             } else {
-                $byline = get_string('postbyxinreplytox', 'hsuforum', array(
+                $byline = get_string('postfromx', 'theme_legend', array(
```

Comment:
This references a string in theme_legend. Move over to hsuforum and remove legend references.